cldf/README.md

```diff
@@ -1,22 +1,23 @@
 <a name="ds-wordlistmetadatajson"> </a>
 
-# Wordlist Concepticon
+# Wordlist CLLD Concepticon as CLDF dataset
 
 **CLDF Metadata**: [Wordlist-metadata.json](./Wordlist-metadata.json)
 
 **Sources**: [sources.bib](./sources.bib)
 
-List, Johann-Mattis and Cysouw, Michael and Forkel, Robert. 2016. Concepticon. A resource for the linking of concept lists. In Chair), Nicoletta Calzolari (Conference and Choukri, Khalid and Declerck, Thierry and Grobelnik, Marko and Maegaard, Bente and Mariani, Joseph and Moreno, Asuncion and Odijk, Jan and Piperidis, Stelios (eds.), Proceedings of the Tenth International Conference on Language Resources and Evaluation, 2393-2400. European Language Resources Association (ELRA).
+The Concepticon is a special Wordlist, where the words are concept labels in particular languages which have been used to elicit lexical data in other languages. These labels are grouped into concept sets, the 'Parameters' of the Concepticon, which can serve as cross-linguistic, comparative concepts.
```

Comment:
I'm guessing that the upper case for "Wordlist" is intentional. I think it makes sense in this context.

Reply:
Yes, `Wordlist` is the name of the CLDF module.

Reply:
I see.